code/modules/clothing/spacesuits/rig_modules/rig_modules.dm

```diff
@@ -299,7 +299,23 @@
 	if(current_capacity >= max_capacity)
 		deactivate()
 
+/obj/item/rig_module/rad_shield/can_install(var/obj/item/clothing/suit/space/rig/target)
+   if(!..())
+      return FALSE
+   if(locate(/obj/item/rig_module/rad_shield/adv) in target.modules) //don't allow both rad mods at once
```

Comment:
it's very strange that this is necessary... this should be handled by the super call; locate(type) should find children; have you tested this?

Reply:
i don't know what you mean by that. what is the super call?
all i know is that the previous locate check doesn't consider descendant objects to be the parent. If it's supposed to, then it's not.
as for tested, yes, i tested it in a local game, trying to install both modules onto each other, with this implementation successfully blocking that happening.